sdk/src/cli/context/get_contexts.rs

```diff
@@ -0,0 +1,78 @@
+use async_trait::async_trait;
+use comfy_table::Table;
+use tracing::{event, Level};
+
+use crate::{
+    cli_command::{CliCommand, PRINT_TARGET},
+    client::Client,
+};
+
+use super::common::{ContextManager};
+
+pub enum GetContextsOutput {
+    Table,
+    List,
+}
+
+pub struct GetContextsCmd {
+    output: GetContextsOutput,
+}
+
+impl GetContextsCmd {
+    pub fn new(output: GetContextsOutput) -> Self {
+        Self { output }
+    }
+
+    fn format_name(name: &str, active_context_key: &str) -> String {
+        if name.eq(active_context_key) {
+            format!("{}*", name)
+        } else {
+            format!("{}", name)
+        }
+    }
+}
+
+impl Default for GetContextsCmd {
+    fn default() -> Self {
+        GetContextsCmd {
+            output: GetContextsOutput::Table,
+        }
+    }
+}
+
+#[async_trait]
+impl CliCommand for GetContextsCmd {
```

Comment:
This command does not need connection to IggyServer so better to implement 
```
    fn login_required(&self) -> bool {
        false
    }
```

Reply:
Ah ok good to know - I had missed this, thank you.